src/gui/guiFormSpecMenu.cpp

```diff
@@ -915,7 +915,10 @@ void GUIFormSpecMenu::parseAnimatedImage(parserData *data, const std::string &el
 
 	auto style = getDefaultStyleForElement("animated_image", spec.fname, "image");
 	e->setNotClipped(style.getBool(StyleSpec::NOCLIP, false));
-	e->drop();
+
+	// Animated images should let events through
+	e->grab();
```

Comment:
why the double grab? By removing the drop and adding a grab, the refs are now 2 times more

Reply:
I don't even know what `grab` does; I just copied from `image`.

Reply:
Grab will increase the reference counter (0 = delete instance). `GUIAnimatedImage::GUIAnimatedImage` already grabs it once, hence not needed here. This code would also cause a memory leak because the data is never freed (counter never reaches 0).